src/accessibility/describe.js

```diff
@@ -16,53 +16,51 @@ const labelTableId = '_labelTable'; //Label Table
 const labelTableElId = '_lte_'; //Label Table Element
 
 /**
- * Creates a screen reader accessible description for the canvas.
- * The first parameter should be a string with a description of the canvas.
- * The second parameter is optional. If specified, it determines how the
- * description is displayed.
+ * Creates a screen reader-accessible description for the canvas.
  *
- * <code class="language-javascript">describe(text, LABEL)</code> displays
- * the description to all users as a <a
- * href="https://en.wikipedia.org/wiki/Museum_label" target="_blank">
- * tombstone or exhibit label/caption</a> in a div
- * adjacent to the canvas. You can style it as you wish in your CSS.
+ * The first parameter, `text`, is the description of the canvas.
  *
- * <code class="language-javascript">describe(text, FALLBACK)</code> makes the
- * description accessible to screen-reader users only, in
- * <a href="https://developer.mozilla.org/en-US/docs/Web/API/Canvas_API/Tutorial/Hit_regions_and_accessibility" target="_blank">
- * a sub DOM inside the canvas element</a>. If a second parameter is not
- * specified, by default, the description will only be available to
- * screen-reader users.
+ * The second parameter, `display`, is optional. It determines how the
+ * description is displayed. If `LABEL` is passed, as in
+ * `describe('A description.', LABEL)`, the description will be visible in
+ * a div element next to the canvas. If `FALLBACK` is passed, as in
+ * `describe('A description.', FALLBACK)`, the description will only be
+ * visible to screen readers. This is the default mode.
+ *
+ * Read
+ * <a href="/learn/labeling-canvases.html">How to label your p5.js code</a> to
+ * learn more about making sketches accessible.
  *
  * @method describe
- * @param  {String} text      description of the canvas
- * @param  {Constant} [display] either LABEL or FALLBACK
+ * @param  {String} text        description of the canvas.
+ * @param  {Constant} [display] either LABEL or FALLBACK.
  *
  * @example
  * <div>
  * <code>
- * describe('pink square with red heart in the bottom right corner');
- * background('pink');
- * fill('red');
- * noStroke();
- * ellipse(67, 67, 20, 20);
- * ellipse(83, 67, 20, 20);
- * triangle(91, 73, 75, 95, 59, 73);
+ * function setup() {
+ *   background('pink');
+ *   fill('red');
+ *   noStroke();
+ *   circle(67, 67, 20);
+ *   circle(83, 67, 20);
+ *   triangle(91, 73, 75, 95, 59, 73);
+ *
+ *   describe('A pink square with a red heart in the bottom-right corner.');
+ * }
  * </code>
  * </div>
  *
  * <div>
  * <code>
- * let x = 0;
  * function draw() {
- *   if (x > 100) {
- *     x = 0;
- *   }
- *   background(220);
+ *   background(200);
+ *
+ *   let x = frameCount % 100;
```

Comment:
It'd be helpful to add a comment here to explain what `frameCount % 100` does in the code